libs/loggers/csv_logger.py

```diff
@@ -55,68 +19,33 @@ def __init__(self, config, camera_id):
-        self.dist_threshold = config.get_section_dict("PostProcessor")["DistThreshold"]
-        os.makedirs(self.objects_log_directory, exist_ok=True)
-        os.makedirs(self.distances_log_directory, exist_ok=True)
 
-    def update(self, frame_number, objects_list, distances):
-        """Write the object and violated distances information of a frame into log files.
-
-        Args: frame_number: current frame number objects_list: A list of dictionary where each dictionary stores
-        information of an object (person) in a frame. distances: A 2-d numpy array that stores distance between each
-        pair of objects.
-        """
-        file_name = str(date.today())
-        objects_log_file_path = os.path.join(self.objects_log_directory, file_name + ".csv")
-        distances_log_file_path = os.path.join(self.distances_log_directory, file_name + ".csv")
-        self.log_objects(objects_list, frame_number, objects_log_file_path)
-        self.log_distances(distances, frame_number, distances_log_file_path)
+        os.makedirs(self.objects_log_directory, exist_ok=True)
 
-    @staticmethod
-    def log_objects(objects_list, frame_number, file_path):
+    def log_objects(self, objects, violating_objects, violating_objects_index_list, violating_objects_count,
+                    detected_objects_cout, environment_score, time_stamp, version):
         """Write objects information of a frame into the object log file.
         Each row of the object log file consist of a detected object (person) information such as
         object (person) ids, bounding box coordinates and frame number.
 
-        Args: objects_list: A list of dictionary where each dictionary stores information of an object (person) in a
-        frame. frame_number: current frame number file_path: log file path
-        """
-        if len(objects_list) != 0:
-            object_dict = list(map(lambda x: prepare_object(x, frame_number), objects_list))
-
-            if not os.path.exists(file_path):
-                with open(file_path, "w", newline="") as csvfile:
-                    field_names = list(object_dict[0].keys())
-                    writer = csv.DictWriter(csvfile, fieldnames=field_names)
-                    writer.writeheader()
-
-            with open(file_path, "a", newline="") as csvfile:
-                field_names = list(object_dict[0].keys())
-                writer = csv.DictWriter(csvfile, fieldnames=field_names)
-                writer.writerows(object_dict)
-
-    def log_distances(self, distances, frame_number, file_path):
-        """Write violated incident's information of a frame into the object log file.
-
-        Each row of the distances log file consist of a violation information such as object (person) ids,
-        distance between these two object and frame number.
-
         Args:
+            objects_list: A list of dictionary where each dictionary stores information of an object (person) in a frame.
```

Comment:
I guess this parameter list should be updated

Reply:
Yes, it was the documentation of the previous function.
Taking into accout that probably we will refactor the loggers I removed the comment for now.
We can document everything after the refactor.